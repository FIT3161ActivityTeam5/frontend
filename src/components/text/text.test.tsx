import React from 'react';
import renderer from 'react-test-renderer';

import Text from './text';

describe('<Text />', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Text>Hello world!</Text>).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('applies styles correctly', () => {
    const tree = renderer.create(
      <Text
        style={{
            fontWeight: '700',
            fontStyle: 'italic',
            fontSize: 28,
            color: 'red',
        }}
      >
          Hello world!
      </Text>
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
