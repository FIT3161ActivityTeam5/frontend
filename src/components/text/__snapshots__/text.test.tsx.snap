// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Text /> applies styles correctly 1`] = `
<Text
  style={
    Array [
      Object {
        "fontFamily": "Poppins_700Bold_Italic",
      },
      Object {
        "color": "red",
        "fontSize": 28,
      },
    ]
  }
>
  Hello world!
</Text>
`;

exports[`<Text /> renders correctly 1`] = `
<Text
  style={
    Array [
      Object {
        "fontFamily": "Poppins_400Regular",
      },
      Object {},
    ]
  }
>
  Hello world!
</Text>
`;
